lua/core/status.lua

```diff
@@ -78,15 +78,15 @@ end
 function M.provider.lsp_client_names(expand_null_ls)
   return function()
     local buf_client_names = {}
-    for _, client in ipairs(vim.lsp.buf_get_clients(0)) do
+    for _, client in pairs(vim.lsp.buf_get_clients(0)) do
       if client.name == "null-ls" and expand_null_ls then
         vim.list_extend(buf_client_names, astronvim.null_ls_sources(vim.bo.filetype, "FORMATTING"))
         vim.list_extend(buf_client_names, astronvim.null_ls_sources(vim.bo.filetype, "DIAGNOSTICS"))
       else
         table.insert(buf_client_names, client.name)
       end
     end
-    return table.concat(buf_client_names, ", ")
+    return table.concat(buf_client_names, ", ") .. " "
```

Comment:
Could you please remove this change in this line as it is not necessary and messes up the padding.

Reply:
ok sure

Reply:
still think padding is better if someone want to have some background color on it